services/migrations/0029_auto_20170711_1725.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
The migration should have a more descriptive name. Supply it with `makemigrations -n <name>`.